src/common/modules/Notifications.js

```diff
@@ -22,13 +22,14 @@ export let SEND = true;
  * @param {string} title the title
  * @param {string} content the message content
  * @param {string[] | string} [substitutions] the message parameters to pass for i18n.getMessage
+ * @param {boolean} [send]
```

Comment:
uhm what does it do? 
Boolean options in methods are generally a bad practice as they are hard to describe. However, especially here, I don't know what this does…

It cancels basically the whole function and does nothing if it is false, but only in _combination_ with `SEND` (it is an `&&` below, is this deliberate?) uhm what? When you don't want a notification to be sent, one would not call this function, would not one?
Can't we remove it?

Reply:
The uppercase `SEND` is for the new option to disable desktop notifications. This lowercase `send` parameter I added at your request above to make it impossible to disable error notifications: https://github.com/rugk/unicodify/pull/80#discussion_r1055888164. This is an optional parameter, so most of the time it will be undefined. When both of these variables are false/undefined it will not "send"/show the notification, otherwise it will.

I agree that this is confusing, but I could not come up with a clearer way to implement your request. Suggestions are of course welcome...

Reply:
Ah, I now get it and quoting myself:
> it should not be able to be disabled, i.e. not controlled by that option, but always be shown.

So the parameter is rather a `optionalNotification = true`, with a description like `If the notification is optional, the user can disable it with an add-on setting.` or so. By default it is optional.
Maybe adding: `If it is not optional, the notification will always be shown. Use the latter notification type only sparsely, e.g. for user-initiated actions, which require feedback to the user.`.

Reply:
Speaking of notification _type_ an alternative would be indeed to introduce an "enum" aka JS Symbol thingy, where you can pass different types/symbols in there, which may be even more readable than a bool parameter.

Reply:
Yes, when the parameter is `true`, the notification is required, but when it is `false`/`undefined` it is optional, which is the default. I could rename it to `requiredNotification` and then add your suggested comments/description. Is that what you are requesting?

Reply:
Yeah that sounds good.

Reply:
OK, will do...